aces/Loris
# [Documentation] Created Survey Accounts module help text

## Brief summary of changes
I created a new help text for the Survey Accounts module, since it didn't have one. I integrated content from @leighevans in 2017 which should have been committed at that time.

#### Testing instructions (if applicable)

1. Ensure all information is correct and easy to read
2. Ensure help text shows up in the LORIS front-end, with correct formatting.

#### Links to related tickets (GitHub, Redmine, ...)

https://github.com/aces/Loris/issues/5241
Original PR was never completed: https://github.com/aces/Loris/pull/3390
